### PR TITLE
Use CRD library instead of schemas

### PR DESCRIPTION
**Pull request by nimalank7**
Description:
- Currently we use CRD schemas which are located in the /schema directory. However updating these is a chore which we'd rather automate
- Using [datreeio](https://github.com/datreeio/CRDs-catalog)'s CRD catalogue will have `kubeconform` download the schemas instead
- Closes [issue 1597](https://github.com/alphagov/govuk-helm-charts/issues/1597)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
           args: >
             -kubernetes-version ${{ env.KUBERNETES_VERSION }}
             -schema-location default
-            -schema-location "./schemas/{{ .ResourceKind }}_{{ .ResourceAPIVersion }}.json"
+            -schema-location "https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json"
```
Comment (sengi) [inline .github/workflows/ci.yml:45]: I think you can fix the line length lint like this:
```suggestion
            -schema-location
            "https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json"
```

File: README.md [hunk 2]
```diff
@@ -76,8 +76,29 @@ You can then run `helm search repo <alias>` to see the charts.
...
+locally.
+
+You can run the validation tests locally by installing `kubeconform` and running
+
+```shell
+mkdir helm-dist
+for c in $(ls charts)
+do
+  helm template "${c}" "charts/${c}" --output-dir helm-dist
+done
```
Comment (sengi) [inline README.md:95]: I know it was already like this, but it didn't work as it was 😄 
```suggestion
for c in charts/*; do
  helm template "$(basename "$c")" "$c" --output-dir helm-dist
done
```

**Review by sengi — APPROVED** [review]
Yay, thanks!

You can drop the issue number from the PR title btw, GitHub doesn't link them and it just makes it harder to read the titles.

File: README.md (hunk 2 above)
Comment (nimalank7) [reply README.md]: Yep fixed!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (nimalank7) [reply .github/workflows/ci.yml]: Fixed by adding a `\` to break the string over into multiple lines.

**Pull request merged**